Margin prop implementation

siddharthkp opened the pull request.

andresgalante commented on core/components/molecules/page-header/page-header.js:
```diff
@@ -40,7 +40,9 @@ const PageHeader = props => {
       </ButtonGroup>
 
-      <Heading size={1}>{props.title}</Heading>
+      <Heading size={1} margin={{ top: 0, bottom: 'xsmall' }}>
```
I suggest we merge this https://github.com/auth0/cosmos/pull/1138 before we move on with margin. Since it changes the way we do spaces on the page header

siddharthkp replied:
Sure thing! added that as a blocker to merge in the description

andresgalante commented:
Sorry @siddharthkp, the page header was out of date from Francos changes. I've just rebased it and push a clean version.

siddharthkp commented:
> Sorry @siddharthkp, the page header was out of date from Francos changes. I've just rebased it and push a clean version.

That's just an example - taken from master. I've added #1138 as a blocker to this PR, that cool?
